### ISSUE TITLE
Highlight our founders on Members page

### ISSUE THREAD

**Issue by danieltott** [Virtual-Coffee/virtualcoffee.io#45]
## Issue Context

I love seeing everyone on our [new Members page](https://virtualcoffee.io/members/), but it's bothering me that Virtual Coffee's founders (@BekahHW and @saramccombs) aren't highlighted in any way. Let's fix that.

## Suggested Solution

Layout-wise, I think maybe just add a header like "Our Founders" or "Our Founding Members" or something to that effect, put Bekah and Sara's blocks in there, and then another header like "Our Awesome Members" and display the rest as they currently are.

```
Our Founders
[bekah] [sara]

Our Members
[member1] [member2]
...
```

## Approach

First step will be to basically duplicate what we have for members inside `src/_data` for founders:

- Create `src/_data/founders` and move `src/_data/members/BekahHW.json` and `src/_data/members/saramccombs.json` into there. 
- Duplicate `src/_data/members.js` and call it `founders.js`
- Adjust code inside `founders.js` where appropriate

This would provide, through the [magic of 11ty javascript data files](https://www.11ty.dev/docs/data-js/), a data object available to our templates called `founders`

Then in `src/members.njk` we would be able to do `{% for founder in founders %}...{% endfor %}`

I'll leave it up to whoever implements this what html to specifically use. Just keep in mind we use [bootstrap](https://getbootstrap.com/docs/4.5/getting-started/introduction/) so all of their classes are available to us if needed.

After getting something together, submitting a PR will create a previewable link, so submitting a PR that may still need some work or with some design questions is very appropriate.

**Comment by ClJarvis** [comment]
Is there anything on their Json that we could use ID them as cofounders? or Would they have to be identified by user name? They only thing I see that says they are cofounders is in their bios that comes from GitHub.

**Comment by dominicduffin1** [comment]
I like the option of putting them at the top under the 'Our Founders' heading, then having 'Our Members' underneath.

**Comment by danieltott** [comment]
@ClJarvis No, we'd definitely have to hard-code their usernames, but that's ok - there's only two of them 😃

**Comment by danieltott** [comment]
Honestly it may be simpler to just create a separate folder and put them in there - create `src/_data/founders` and move `src/_data/members/BekahHW.json` and `src/_data/members/saramccombs.json` into there. Then duplicate `src/_data/members.js` and call it `founders.js`

This would provide, through the [magic of 11ty javascript data files](https://www.11ty.dev/docs/data-js/), a data object available to our templates called `founders`

then in `src/members.njk` we would be able to do `{% for founder in founders %}`

**Comment by ClJarvis** [comment]
That make sense. That prevents having to check if a member is or isn't  a fonder.

**Comment by danieltott** [comment]
Ok - I added that approach to the description. Consider this one available!

**Comment by ClJarvis** [comment]
I would be interested in working on this.

**Comment by BekahHW** [comment]
@ClJarvis you got it! Let us know if you have any questions.

**Comment by danieltott** [comment]
Closed via #47 - Thank you @ClJarvis !
